jdaviz/configs/imviz/helper.py

```diff
@@ -29,6 +29,40 @@ def default_viewer(self):
         """Default viewer instance. This is typically the first viewer ("imviz-0")."""
         return self._default_viewer
 
+    def create_image_viewer(self):
+        """Create a new image viewer.
+
+        To display data in this new viewer programmatically,
+        first get the new viewer ID from the small tab on the top
+        left of viewer display. Then, use
+        :meth:`~jdaviz.app.Application.add_data_to_viewer` from ``imviz.app``
+        by passing in the new viewer ID and the desired data label,
+        once per dataset you wish to display.
+
+        Alternately, you can also display data interactively via the GUI.
+
+        """
+        from jdaviz.configs.imviz.plugins.viewers import ImvizImageView
+
+        # Cannot assign data to real Data because it loads but it will
+        # not update checkbox in Data menu.
+        self.app.hub.broadcast(NewViewerMessage(ImvizImageView, data=None, sender=self.app))
```

Comment:
Can this return the viewer_id?  Or the viewer instance?  It would be much more convenient for the user to be able to do wither:
```
id = app.create_image_viewer()
app.get_viewer(id).show_data(<whatever>)
```
or better yet
```
app.create_image_viewer().show_data(<whatever>)
```

Reply:
This would require me to invoke this directly instead of using a more canon call that is `self.app.hub.broadcast(NewViewerMessage(...))`. I don't know if doing this would have side effects, but I can play with it and see.

Reply:
Is this because you have to wait for the hub to do something async? Is there maybe a way to just wait for it?  Or return the id without the reference?

Reply:
(Or the answer could be "need to phone a friend named @astrofrog" :wink:)

Reply:
`viewer.show_data()` does not exist. If you want that, it would be a new feature request. But I'll update the PR shortly for your other two earlier requests.

Reply:
It now returns the viewer instance. I don't see any bad side effects but if you see it, please let me know. See the updated example notebook in this PR.